js/src/dapps/registry/reducers.js

```diff
@@ -1,13 +1,23 @@
-import lookupReducer from './Lookup/reducers.js';
+import accountsReducer from './accounts/reducers.js';
+import lookupReducer from './lookup/reducers.js';
+import eventsReducer from './events/reducers.js';
+import registerReducer from './register/reducers.js';
 
 const initialState = {
+  accounts: accountsReducer(undefined, { type: '' }),
   contract: null,
   fee: null,
   owner: null,
-  lookup: lookupReducer(undefined, { type: '' })
+  lookup: lookupReducer(undefined, { type: '' }),
+  events: eventsReducer(undefined, { type: '' }),
+  register: registerReducer(undefined, { type: '' })
 };
 
 export default (state = initialState, action) => {
+  if (action.type.slice(0, 8) === 'accounts') {
```

Comment:
It doesn't have to be an array `action.type.indexOf('accounts') === 0` is ok, but you could wrap it into function instead of doing some string operations in logic code - imho it could help with readability.